path_template.go

```diff
@@ -1,276 +0,0 @@
-// Copyright 2016, Google Inc.
```

Comment:
I think removing path_template.go is not in the scope of this change, right?
This implementation is incomplete but probably it would be better to keep it until you refine / reimplement it.